app/src/page/wallet/index.js

```diff
@@ -1,4 +1,14 @@
 import { connect } from 'react-redux';
+import { doSetClientSetting } from '../../redux/actions/settings';
+import { makeSelectClientSetting } from '../../redux/selectors/settings';
 import WalletPage from './view';
 
-export default connect(null, null)(WalletPage);
+const select = state => ({
+  understandsRisks: makeSelectClientSetting("alphaUnderstandsRisks")(state),
```

Comment:
"alphaUnderstandsRisks" should be a constant

Reply:
I was trying to avoid having to modify `lbry-redux` since this is meant to be temporary. I'll create a new file for mobile-specific constants.